### PR TITLE
refactor: lint and edit examples folder

### PR DESCRIPTION
**Pull request by codeman1o1**
## Summary

This PR refactors files in the `example` folder.
The following tools were used:
- Pylint
- Flake8
- Black
- Sourcery

Only the `examples` folder has been refactored, so that is why the documentation checkmark isn't checked
I did not run `task pyright` because I have `pylance` and `VS Code` so that is why it isn't checked

## Checklist

- [x] If code changes were made then they have been tested.
  - [ ] I have updated the documentation to reflect the changes.
  - [ ] I have run `task pyright` and fixed the relevant issues.
- [ ] This PR fixes an issue.
- [ ] This PR adds something new (e.g. new method or parameters).
- [ ] This PR is a breaking change (e.g. methods or parameters removed/renamed)
- [ ] This PR is **not** a code change (e.g. documentation, README, ...)

### CODE REVIEW

**Comment by ooliver1** [comment]
please could you sort out the conflicts? black and isort should have already been used on the examples due to #627 
image

**Comment by codeman1o1** [comment]
That commit should hopefully fix some conflicts

**Comment by DenverCoder1** [comment]
A lot of these single quotes changed were apostrophes in comments and docstrings, which should not have been changed to double quotes.

**Comment by codeman1o1** [comment]
I reverted back to 14726d8371f9360dd15a06d75bff3a75c3db9df5

**Comment by codeman1o1** [comment]
Am I now good to go to resolve the conflicts?

File: examples/guessing_game.py [hunk 1]
```diff
@@ -10,17 +10,23 @@
...
+        return message.author == ctx.author and message.content.isdigit()
+
+    answer = random.randin(1, 10)
 
-    try:
-        guess = await bot.wait_for("message", check=is_correct, timeout=5.0)
-    except asyncio.TimeoutError:
-        return await ctx.send(f"Sorry, you took too long it was {answer}.")
+    if not guess:
+        await ctx.send('"uess a number between 1 and 10.")
```
Comment (ooliver1) [inline examples/guessing_game.py:22]:
```suggestion
        await ctx.send("Guess a number between 1 and 10.")
```#my bad

File: examples/guessing_game.py [hunk 2]
```diff
@@ -10,17 +10,24 @@
...
+@bot.command()
+async def guess(ctx, guess: int = None):
+    def is_correct(message):
+        return message.author == ctx.author and message.content.isdigit()
+
+    answer = random.randin(1, 10)
+
+    if not guess:
+        await ctx.send("Guess a number between 1 and 10.")
+```#my bad
```
Comment (ooliver1) [inline examples/guessing_game.py:23]:
```suggestion
        await ctx.send("Guess a number between 1 and 10.")
```
what the fuck github

**Review by Skelmis — APPROVED** [review]

**Review by teaishealthy — CHANGES_REQUESTED** [review]

**Review by teaishealthy — APPROVED** [review]

**Review by ooliver1 — APPROVED** [review]

**Pull request merged**